consensus/tests/integration_test.rs

```diff
@@ -137,3 +137,18 @@ async fn basic_1() {
     }
     serve_task.await.unwrap();
 }
+
+/// Tendermint lock happens and it helps to keep the safety by reaching the consensus in the second round.
+#[ignore]
+#[tokio::test]
+async fn lock_1() {}
```

Comment:
will refer [this](https://github.com/postech-dao/simperby/pull/486) to implement `lock_1` test and will take this